src/theory/arith/congruence_manager.h

```diff
@@ -101,6 +102,16 @@ class ArithCongruenceManager {
 
   eq::EqualityEngine d_ee;
 
+  /** proof manager */
+  ProofNodeManager* d_pnm;
+  // A proof generator for storing proofs of facts that are asserted to the EQ
+  // engine. Note that these proofs **are not closed**, and assume the
+  // explanation of these facts. This is why this generator is separate from the
+  // TheoryArithPrivate generator, which stores closed proofs.
+  std::unique_ptr<EagerProofGenerator> d_pfGenEe;
+  // A proof generator for TrustNodes sent to the theory.
```

Comment:
I'd be curious to see more documentation on this.  Do you mean TrustNodes sent to this class? (the main Theory class doesn't input TrustNode).

Reply:
Done in ef7ba20c3.